packages/coq-serapi/coq-serapi.8.11.0+0.11.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+maintainer:   "e@x80.org"
+homepage:     "https://github.com/ejgallego/coq-serapi"
+bug-reports:  "https://github.com/ejgallego/coq-serapi/issues"
+dev-repo:     "git+https://github.com/ejgallego/coq-serapi.git"
+license:      "GPL-3.0-or-later"
+doc:          "https://ejgallego.github.io/coq-serapi/"
+
+synopsis:     "Serialization library and protocol for machine interaction with the Coq proof assistant"
+description:  """
+SerAPI is a library for machine-to-machine interaction with the
+Coq proof assistant, with particular emphasis on applications in IDEs,
+code analysis tools, and machine learning. SerAPI provides automatic
+serialization of Coq's internal OCaml datatypes from/to JSON or
+S-expressions (sexps).
+"""
+
+authors: [
+  "Emilio Jesús Gallego Arias"
+  "Karl Palmskog"
+  "Clément Pit-Claudel"
+  "Kaiyu Yang"
+]
+
+depends: [
+  "ocaml"               {           >= "4.07.0"            }
+  "coq"                 { !pinned & >= "8.11.0" & < "8.12" }
```

Comment:
Apparently opam does not support `!pinned` (but does `pinned`):
```
[ERROR] undefined filter variable in dependencies of coq-serapi.8.11.0+0.11.0: pinned
```

Reply:
Thanks @kit-ty-kate , I'm always confused about this, as for example this does get thru our own CI. Will amend the opam file.

Reply:
Our CI uses `opam update --strict` which prevents such errors from being put under the rug (by default opam ignores such errors)